packages/runtime/container-runtime/src/deltaScheduler.ts

```diff
@@ -86,7 +90,18 @@ export class DeltaScheduler {
 
                 // eslint-disable-next-line @typescript-eslint/no-floating-promises
                 this.deltaManager.inbound.pause();
+
                 setTimeout(() => {
+                    if (this.schedulingLog) {
+                        this.logger.sendTelemetryEvent({
+                            eventName: "DeltaManagerPaused",
+                            duration: elapsedTime,
+                            numberOfOps: this.deltaManager.inbound.length,
```

Comment:
We should add all state tracking for logging to`this.schedulingLog` or remove it and track these independently. Right now, its a mix and it adds confusion.
Instead of initializing the `schedulingLog` in `batchEnd` we should do it in `batchBegin` and assign the current inbound queue length to it. And then use it as a reference to log the number of ops processed, remaining, etc. whenever we want to log.

Reply:
We don't use queueLengthBeforeBatchBegin if this.schedulingLog is undefined, so it would make sense for it to be part of the schedulingLog itself.

Reply:
yes, I removed queueLengthBeforeBatchBegin - the firstSequenceNumber and lastSequenceNumbers would give us the number of ops processed, isn't it ?